server.js

```diff
@@ -48,11 +47,15 @@ async function main() {
         res.send(doc)
     })
 
+    app.get("/dashboard", function(_, res) {
```

Comment:
Create a check to make sure the current user is an admin. This way, only admins can be served the dashboard page.